src/java.desktop/share/classes/javax/swing/JPopupMenu.java

```diff
@@ -1401,6 +1427,11 @@ private void readObject(ObjectInputStream s)
             popup = (Popup)values.elementAt(++indexCounter);
             indexCounter++;
         }
+        if(indexCounter < maxCounter && values.elementAt(indexCounter).
```

Comment:
```suggestion
        if (indexCounter < maxCounter && values.elementAt(indexCounter).
```

Reply:
the formatting is consistent with the previous 2 checks so I didn't change, so either I have to change the previous lines formatting too since it will look odd if we change mine and not the previous so I kept it as it is like previous check.